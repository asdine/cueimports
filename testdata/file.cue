package file

wall: {
	color: colors.#Red
	size:  dimensions.#Dimensions & {
		w: 100
		h: 100
	}
	altSize: alt.#AltDimensions & {
		w: 150
		h: 150
	}
	priceM2: math.Floor(size.w * size.h / 10000)
	objects: [
		shapes.#Square & {
			d: {
				w: 100
				h: 100
			}
		},
	]
}

floor: {
	color: colors.#Color
}

enc: json.Marshal(wall)

ceiling: {
	color: default.color
}
