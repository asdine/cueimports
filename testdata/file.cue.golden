package file

import (
	"encoding/json"
	"math"
	
	"colors"
	"test.com/dimensions"
	"test.com/dimensions:alt"
)

wall: {
	color: colors.#Red
	size:  dimensions.#Dimensions & {
		w: 100
		h: 100
	}
	altSize: alt.#AltDimensions & {
		w: 150
		h: 150
	}
	priceM2: math.Floor(w * h / 10000)
}

floor: {
	color: colors.#Color
}

enc: json.Marshal(wall)

ceiling: {
	color: default.color
}
